apps/test/unit/turtle/turtleTest.js

```diff
@@ -90,6 +90,26 @@ describe('Artist', () => {
     });
   });
 
+  describe('Accepts a third argument parameter', () => {
+    it('draws a sticker when size is null', () => {
+      let artist = new Artist();
+      let sticker = new Image(100, 100);
+      let img = sticker;
```

Comment:
small nit: `sticker` isn't used.
combine line 96+97 to `const img = new Image(100, 100);`

also if you don't modify the variable, use `const` instead of `let`

Reply:
http://wesbos.com/let-vs-const/